Doc/library/math.rst

```diff
@@ -166,6 +166,20 @@ Number-theoretic and representation functions
    Return ``True`` if *x* is a NaN (not a number), and ``False`` otherwise.
 
 
+.. function:: isqrt(n)
+
+   Return the "integer square root" of the integer *n*. This is the integer
+   part of the exact square root of *n*.
+
+   For some applications, it is convenient to have the ceiling of the square
+   root of *n* (the smallest integer ``a`` such that ``n <= a * a``) rather
```

Comment:
If you prefer more typographic display of the formula:
```suggestion
   root of *n* (the smallest integer *a* such that *n* ≤ *a*\ ²) rather
```
If you prefer this, it would be worth to use also non-breakable spaces around the ≤ operator: https://stackoverflow.com/questions/11830242/how-can-i-make-a-non-breaking-space-in-restructuredtext